swift/swift.py

```diff
@@ -173,6 +173,26 @@ def load(self, appInfos: Mapping[str, AppInfo]):
         for name, info in appInfos.items():
             self.createApp(name, info)
 
+    def addFrame(self, name: str, frame: QWidget, info: AppInfo):
+        """Adds a frame of the app and wraps it with a dock widget.
+        

```

Comment:
It would be better to note:
`This is not a swift-call because QWidget is not Serializable.`